File: Java/q1048.java
// Aumento de Salário

import java.util.Scanner;

public class Main {
 
    public static void main(String[] args) {
 
       Scanner input = new Scanner(System.in);
       double sal = input.nextDouble();
       double perc;

       if (sal <= 400){
           perc = 15.0;
       } else if (sal <= 800) {
           perc = 12.0;
       } else if (sal <= 1200) {
           perc = 10.0;
       } else if (sal <= 2000) {
           perc = 7.0;
       } else {
           perc = 4.0;
       }
       
       double reajuste = perc/100 * sal;
       double novoSalario = sal + reajuste;
       
       System.out.printf("Novo salario: %.2f%n", novoSalario);
       System.out.printf("Reajuste ganho: %.2f%n", reajuste);
       System.out.printf("Em percentual: %.0f %%%n", perc);
    }
}

File: Cpp/q1048.cpp
#include <iostream>
#include <iomanip>
using namespace std;

int main(){

  double valor, ganho, novosalario;
  int reajuste;
  cin >> valor;
  
  ganho = 0;
  novosalario = 0;

  if(valor >= 0 && valor <= 400.00){ 
     reajuste = 15; 
  } else if(valor > 400.00 && valor <= 800.00){
     reajuste = 12; 
  } else if(valor > 800.00 && valor <= 1200.00){
     reajuste = 10;
  } else if(valor > 1200.00 && valor <= 2000.00){
     reajuste = 7;
  } else {
     reajuste = 4;
  }

  ganho = (reajuste*valor/100);
  novosalario = (reajuste*valor/100)+valor;

  cout << "Novo salario: " << fixed << setprecision(2) << novosalario << endl;
  cout << "Reajuste ganho: " << fixed << setprecision(2) << ganho << endl;
  cout << "Em percentual: " << reajuste << " %" << endl;

  return 0;
}
    